### PR TITLE
Upgrade to `postcss` 5

### LINKED ISSUE #5: eachDecl, eachRule and removeSelf are deprecated
According to the warning generated:

```
Container#eachDecl is deprecated. Use Container#walkDecls instead.
Container#eachRule is deprecated. Use Container#walkRules instead.
Node#removeSelf is deprecated. Use Node#remove.
```

### PR DESCRIPTION
**Pull request by reitermarkus**
Fixes https://github.com/yisibl/postcss-unprefix/issues/5.

### CODE REVIEW

File: test/index.js [hunk 1]
```diff
@@ -26,9 +24,9 @@ function compareFixtures(t, name, msg, opts, postcssOpts) {
 }
 
 test("unprefix", function(t) {
-  // compareFixtures(t, "one", "必须生成新的不带前缀的标准属性")
+  compareFixtures(t, "one", "必须生成新的不带前缀的标准属性")
   compareFixtures(t, "flex", "必须生成新的不带前缀的标准属性")
-  // compareFixtures(t, "double", "必须生成新的不带前缀的标准属性")
-  compareFixtures(t, "linear-gradient", "必须生成新的不带前缀的标准属性")
+  compareFixtures(t, "double", "必须生成新的不带前缀的标准属性")
+  // compareFixtures(t, "linear-gradient", "必须生成新的不带前缀的标准属性")
```
Comment (TrySound) [inline test/index.js:30]: Why did you comment this?

Comment (reitermarkus) [inline test/index.js:30]: I commented this because it fails due to `postcss-gradientfixer` and not `unprefix`.

**Comment by reitermarkus** [comment]
Push.

**Comment by yisibl** [comment]
Thanks!

**Comment by reitermarkus** [comment]
No problem. =)

**Pull request merged**